streaming/base/converters/dataframe_to_mds.py

```diff
@@ -33,41 +33,42 @@
 logger = logging.getLogger(__name__)
 
 MAPPING_SPARK_TO_MDS = {
-    ByteType: 'uint8',
-    ShortType: 'uint16',
-    IntegerType: 'int',
-    LongType: 'int64',
-    FloatType: 'float32',
-    DoubleType: 'float64',
-    DecimalType: 'str_decimal',
-    StringType: 'str',
-    BinaryType: 'bytes',
-    BooleanType: None,
-    TimestampType: None,
-    TimestampNTZType: None,
-    DateType: None,
-    DayTimeIntervalType: None,
-    ArrayType: None,
-    MapType: None,
-    StructType: None,
-    StructField: None
+    ByteType(): 'uint8',
+    ShortType(): 'uint16',
+    IntegerType(): 'int',
+    LongType(): 'int64',
+    FloatType(): 'float32',
+    DoubleType(): 'float64',
+    DecimalType(): 'str_decimal',
+    StringType(): 'str',
+    BinaryType(): 'bytes',
+    BooleanType(): None,
+    TimestampType(): None,
+    TimestampNTZType(): None,
+    DateType(): None,
+    DayTimeIntervalType(): None,
+    ArrayType(IntegerType()): 'ndarray:int',
```

Comment:
I figure the `ArrayType(IntegerType())` should go between the ShortType() and the LongType() ones?

Reply:
Yes. IntegerType is 4 bytes, same as python's int? 
short=2bytes, and long = 8bytes.

Reply:
python `int` is actually a bigint (unbounded)

but MDS `int` is stored in a `int64`

believe this is a `LongType()` then?

Reply:
oh sorry I was thinking backwards, all good lmao